ui/app/AppLayouts/Communities/stores/CommunitiesStore.qml

```diff
@@ -65,12 +72,15 @@ QtObject {
                                 },
                                 bannerJsonStr: ""
                              }) {
-        return communitiesModuleInst.createCommunity(
+        const result = communitiesModuleInst.createCommunity(
                     args.name, args.description, args.introMessage, args.outroMessage, args.options.checkedMembership,
                     args.color, args.tags,
                     args.image.src, args.image.AX, args.image.AY, args.image.BX, args.image.BY,
                     args.options.historyArchiveSupportEnabled, args.options.pinMessagesAllowedForMembers,
                     args.bannerJsonStr, args.options.encrypted);
+        if (result)
+            settings.firstCommunityCreated = true
```

Comment:
I think there was a misunderstanding in the issue, we should remove the badge when we open the popup. Not everyone needs to create a community :smile: 

> Upon clicking the “Create New Community” button, the new feature badges are removed.

https://www.figma.com/design/qHfFm7C9LwtXpfdbxssCK3/Kuba%E2%8E%9CDesktop---Communities?node-id=4381-387424&p=f&t=MkGaH5Nx0Pyzn6B6-0

Reply:
Ah ok, I'll update the logic

Reply:
Fixed :)

https://github.com/user-attachments/assets/542aff67-c7ec-4a4f-8495-af870b27bc14

Reply:
Testing it I see that when I create a new account the badge doesn't appear:

https://github.com/user-attachments/assets/72fbb10f-f1d9-466b-9f4a-23c05175ea71

Reply:
> Testing it I see that when I create a new account the badge doesn't appear:

Ah right, the setting is indeed per installation; I can change it to per user if needed

Reply:
Yes, I think this is the expected behavior. Asked @benjthayer and he confirmed!

Reply:
Perfect! I tested it again and it works as expected!